torch/_inductor/utils.py

```diff
@@ -533,7 +533,7 @@ def clear_cache(self: Any) -> None:
     return wrapper  # type: ignore[return-value]
 
 
-def aggregate_origins(
+def _aggregate_origins(
```

Comment:
why are we prefixing these with underscores ? just wondering

Reply:
Gah, you're absolutely right, I shouldn't fiddle. I'll clean away all these of unnecessary tweaks.